File: TEST data for large repo/repo-Copy5/Blueprism/BluePrism.Api/BluePrism.Api.Domain/ScheduledTask.cs
namespace BluePrism.Api.Domain
{
    public class ScheduledTask
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public bool FailFastOnError { get; set; }
        public int DelayAfterEnd { get; set; }
        public int OnSuccessTaskId { get; set; }
        public string OnSuccessTaskName { get; set; }
        public int OnFailureTaskId { get; set; }
        public string OnFailureTaskName { get; set; }
    }
}

File: TEST data for large repo/Repo Copy-3/Blueprism/BluePrism.Server.Domain.Models/ScheduledTask.vb
Imports System.Runtime.Serialization

<Serializable()>
<DataContract([Namespace]:="bp")>
Public Class ScheduledTask
    <DataMember(Name:="id", EmitDefaultValue:=False)>
    Public Property Id As Integer
    <DataMember(Name:="n", EmitDefaultValue:=False)>
    Public Property Name As String
    <DataMember(Name:="d", EmitDefaultValue:=False)>
    Public Property Description As String
    <DataMember(Name:="f", EmitDefaultValue:=False)>
    Public Property FailFastOnError As Boolean
    <DataMember(Name:="d", EmitDefaultValue:=False)>
    Public Property DelayAfterEnd As Integer
    <DataMember(Name:="osid", EmitDefaultValue:=False)>
    Public Property OnSuccessTaskId As Integer
    <DataMember(Name:="osn", EmitDefaultValue:=False)>
    Public Property OnSuccessTaskName As String
    <DataMember(Name:="ofid", EmitDefaultValue:=False)>
    Public Property OnFailureTaskId As Integer
    <DataMember(Name:="ofn", EmitDefaultValue:=False)>
    Public Property OnFailureTaskName As String
End Class

